Geometry/CaloEventSetup/interface/CaloGeometryDBEP.h

```diff
@@ -207,9 +244,28 @@ class CaloGeometryDBEP : public edm::ESProducer
       }
     
 private:
+      std::tuple<const Alignments *, const Alignments *> getAlignGlobal(const typename T::AlignedRecord& iRecord) const {
+	 const Alignments* alignPtr  ( nullptr ) ;
+	 const Alignments* globalPtr ( nullptr ) ;
+         if constexpr (calogeometryDBEPimpl::HasAlignmentRecord<T>::value) {
+	    if( m_applyAlignment ) // get ptr if necessary
+	    {
+	       const auto& alignments = iRecord.get( alignmentTokens_.alignments ) ;
+	       // require expected size
+	       assert( alignments.m_align.size() == T::numberOfAlignments() ) ;
+	       alignPtr = &alignments ;
+
+	       const auto& globals = iRecord.get( alignmentTokens_.globals ) ;
+	       globalPtr = &globals ;
+	    }
+	 }
+         return std::make_tuple(alignPtr, globalPtr);
+      }
 
+    typename calogeometryDBEPimpl::AlignmentTokens<T> alignmentTokens_;
```

Comment:
@makortel - it would be nice not to mix ```m_variableName``` with ```variableName_```. Could you, please, choose one or another convention?

Reply:
Thanks for pointing it out. I usually try to follow existing conventions, but in this case I went with the trailing underscore as that's what our coding rules suggest (2.8) (and it is less work for me).